pywinauto/application.py

```diff
@@ -307,7 +309,7 @@ def __getitem__(self, key):
         """
         # if we already have 2 levels of criteria (dlg, control)
         # then resolve the control and do a getitem on it for the
-        if len(self.criteria) >= 2:
+        if len(self.criteria) >= 2:  # FIXME - this is surprising
```

Comment:
This constraint should be for `backend="win32"` only. We missed additional check of backend name here. Thanks for catching it! We may discuss it in separate issue though.

Reply:
Oh, nevermind, this is something different. Anyway let's discuss it later.